modules/core_project_factory/main.tf

```
@@ -123,7 +123,7 @@ resource "google_resource_manager_lien" "lien" {
   APIs configuration
  *****************************************/
 resource "google_project_service" "project_services" {
-  count = "${length(var.activate_apis)}"
+  count = "${ var.apis_authority == "true" ? 0 : length(var.activate_apis)}"
```

This expression can be simplified:

```suggestion
  count = "${var.apis_authority ? 0 : length(var.activate_apis)}"
```

---

Thank you for your review. Done.